install/generator/03-syndesis-ui.yml.mustache

```diff
@@ -127,21 +127,15 @@
         },
 {{#Productized}}
         "branding": {
-          "logoWhiteBg": "",
-          "logoDarkBg": "",
-          "iconWhiteBg": "assets/images/FuseOnlineLogo_Black.svg",
-          "iconDarkBg": "assets/images/FuseOnlineLogo_White.svg",
```

Comment:
And I think this line too.  This is the bit that is relevant when you set `productBuild` there to `true`.

And yes, I know it's convoluted, I'm honestly not sure why :-)

Reply:
Repeating the same set here:

```
"logoWhiteBg": "",
"logoDarkBg": "",
"iconWhiteBg": "assets/images/FuseOnlineLogo_Black.svg",
"logoDarkBg": "assets/images/syndesis_logo_icon_darkbkg.png",
```